.github/workflows/pr.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn --frozen-lockfile
+      - run: yarn build
```

Comment:
We need it because if you look closely, the tests are importing the watcher from `./dist/lib/watcher`. This is because we need the runner to be compiled to a js file so that we can fork it (`fork('runner.js')`). We cannot fork a .ts file.
We could maybe just transpile that file but I figured it just made more sense to compile the project

Reply:
Gotchya

Could you add a comment about this to DEVELOPMENT.md, since it means (unit) tests will now need a build step before being run.

If you can find a way to avoid needing this that'd be good, but only if its less evil of  solution :D

Reply:
The solution I see would be to `transpileModule('runner.ts')` in a jest `beforeAll()` hook, and probably delete that file in a `afterAll()`

Reply:
Seems reasonable